data/normalizer/standard_test.go

```diff
@@ -36,9 +36,9 @@ var _ = Describe("Standard", func() {
 			var err error
 			standardContext, err = context.NewStandard(test.NewLogger())
 			Expect(standardContext).ToNot(BeNil())
-			Expect(err).ToNot(HaveOccurred())
+			Expect(err).To(Succeed())
```

Comment:
Yes, `To(Succeed())` is the same as `ToNot(HaveOccurred())` and both mean that the error is nil.